backend/wmg/data/validation/validation.py

```diff
@@ -103,7 +103,7 @@ def validate_cube_size(self):
             )
 
     def validate_cube_species(self):
-        with tiledb.open(self.expression_summary_path, "r") as cube:
```

Comment:
no need to load expression summary into memory for this - cell counts cube has all the necessary data.